Themes/default/index.template.php

```diff
@@ -212,22 +212,30 @@ function template_body_above()
 		if (!empty($context['user']['avatar']))
 			echo $context['user']['avatar']['image'];
 
-		echo $context['user']['name'], '</a>
+		echo '<span class="textmenu">', $context['user']['name'], '</span></a>
 					<div id="profile_menu" class="top_menu"></div>
 				</li>';
 
 		// Secondly, PMs if we're doing them
 		if ($context['allow_pm'])
 			echo '
 				<li>
-					<a href="', $scripturl, '?action=pm"', !empty($context['self_pm']) ? ' class="active"' : '', ' id="pm_menu_top">', $txt['pm_short'], !empty($context['user']['unread_messages']) ? ' <span class="amt">' . $context['user']['unread_messages'] . '</span>' : '', '</a>
+					<a href="', $scripturl, '?action=pm"', !empty($context['self_pm']) ? ' class="active"' : '', ' id="pm_menu_top">
+						<span class="main_icons inbox"></span>
```

Comment:
use mail icon instead?

Reply:
I think using the inbox icon here is more consistent with the rest of the relevant UI. We use the inbox icon for the full PM inbox, and this menu popup simply provides a view of a subset of items in that inbox.